app/models/service_template.rb

```diff
@@ -242,4 +242,8 @@ def validate_template
       r.respond_to?(:template_valid?) && !r.template_valid?
     end.try(:resource).try(:validate_template) || {:valid => true, :message => nil}
   end
+
+  def validate_order
+    service_template_catalog && display
```

Comment:
correct, the API dynamically calls these (similar to the vm subsystem style, validate_start, ...).
I'll create an orderable? alias and repush.  Thanks.